movo_common/si_utils/src/si_utils/jaco_jtas_test.py

```diff
@@ -117,7 +117,8 @@ def clear(self, arm='right'):
 def main():
     rospy.init_node('jaco_jtas_test')
     arm_laterality = 'right'
-    dof = rospy.get_param('/' + arm_laterality + '_arm_ctl/jaco_dof')
+    #dof = rospy.get_param('/' + arm_laterality + '_arm_ctl/jaco_dof')
+    dof = '6dof'
```

Comment:
this should be undone

Reply:
Apologies again, this was for debugging purposes